modules/upgrade/partials/rolling-upgrades/upgrade-limitations.adoc

```diff
@@ -22,4 +22,4 @@ Upgrade the Remote Read Replica cluster first, ensuring it's on the same version
 When upgrading to Redpanda 23.2, metadata from object storage is not synchronized until all brokers in the cluster are upgraded. If you need to force a mixed-version cluster to sync read replicas, transfer partition leadership to brokers running the original version.
 
 * *Controller snapshots*:
-xref:get-started:architecture.adoc#controller-partition-and-snapshots[Controller snapshots] are disabled in upgraded clusters. To re-enable them, contact link:https://support.redpanda.com/hc/en-us[Redpanda Support^].
\ No newline at end of file
+Starting in Redpanda version 23.2, when older clusters are upgraded to a version supporting controller snapshots, this feature is enabled by default.  After controller snapshots is enabled, it persists across cluster restarts and future upgrades. New clusters created subsequently automatically enable controller snapshots.
```

Comment:
Is this the final wording? Still a bit confusing to me.

> Starting in Redpanda version 23.2, when older clusters are upgraded to a version supporting controller snapshots, this feature is enabled by default

First, this is true for 23.3, not 23.2. Second, "a version supporting controller snapshots" _is_ 23.2 and later versions. The text hints that they are somehow different.

> New clusters created subsequently automatically enable controller snapshots.

Subsequently to what? The previous sentence talks about enabling controller snapshots so it is probably that. But you can't enable anything without a cluster which is a contradiction with this cluster being new.